contribute.md

```diff
@@ -51,17 +51,16 @@ If you experience problems related to therubyracer or libv8, you can install wit
 
 ### Setup test environment
 1. Copy `config/database.yml.example` to `config/database.yml`
-2. Edit `config/settings.yaml` to set :locations_enabled: and :organizations_enabled: to true
-3. Create your database: `rake db:migrate`
-4. Run all the tests: `rake test`
-5. Or a single test: `ruby -Itest test/functional/your_test.rb` (if you didn't run all tests before, you need to prepare the test environment with `rake db:test:prepare` the first time).
+2. Create your database: `rake db:migrate`
+3. Run all the tests: `rake test`
+4. Or a single test: `ruby -Itest test/functional/your_test.rb` (if you didn't run all tests before, you need to prepare the test environment with `rake db:test:prepare` the first time).
```

Comment:
Should we use Markdown's feature of using "1." for every numbered line? It makes these diffs smaller, but might make less sense if people ever read these files directly....